fmriprep/workflows/bold/base.py

```diff
@@ -796,18 +797,19 @@ def init_func_preproc_wf(bold_file, ignore, freesurfer,
         ])
 
         # CIFTI output
-        if cifti_output and 'template' in output_spaces:
+        if cifti_output and surface_spaces:
```

Comment:
We still need there to be a `'template'` (or whatever it's changing to) output space for the volumetric portion, correct?

Reply:
What about we add the volumetric reference to the output_spaces list at parameter parsing (like we do with AROMA)?

For the current implementation, we would add `'template'`. With poldracklab/smriprep#37 this will probably mean adding something like `'fsLR'` instead.

WDYT?

Reply:
I don't really understand the proposal.

Reply:
Sorry.

The reason we checked `'template' in output_spaces` here is that we needed the volumetric segmentation to extract the BOLD signal. The check ensures that the registration to MNI was performed and the aseg resampled into native space.

What I'm proposing is that we do as with AROMA: https://github.com/poldracklab/fmriprep/blob/84672ee6e171d84f06bb271dbc7bb46d2b20cc2f/fmriprep/cli/run.py#L575-L582 If the command line argument does not include `'template'` but `--cifti-output` is present, then we add `'template'` to the `output_spaces` right after argument parsing.

In the future (referenced PR to smriprep above) we will likely have a new `'fsLR`' option for templates, and we probably want to bind the `--cifti-output` option to the HCP template directly. But that is a different story I shouldn't have introduced here, probably.

Reply:
Oh, sure.